tests/phpunit/integration/Core/Util/REST_Entity_Search_ControllerTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * REST_Entity_Search_ControllerTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Util
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\REST_API\REST_Routes;
+use Google\Site_Kit\Core\Util\REST_Entity_Search_Controller;
+use Google\Site_Kit\Tests\TestCase;
+use WP_REST_Request;
+use WP_REST_Server;
+
+class REST_Entity_Search_ControllerTest extends TestCase {
```

Comment:
It would be good to see test(s) which verify the behaviour of the endpoint, i.e. check the correct entities are returned for a search.

Reply:
Added two more tests. Cheers.